service/src/main/java/uk/gov/hmcts/reform/fpl/service/InboxLookupService.java

```diff
@@ -39,6 +43,12 @@
                 .ifPresent(recipients::add);
         }
 
+        if (!request.isExcludeLegalRepresentatives()) {
```

Comment:
Do we want this after the fallback inbox or not? Can't remember the exact reason for the fallback stuff but if there is now a legal rep but no la/solicitor email then the fallback won't be added.

Reply:
checked with gail, she's ok with this. Counting the rep as a 1 in case of fallback